### ISSUE TITLE
Unfinished Deliverables

### ISSUE THREAD

**Issue by huntrben** [CIS-499-Group3/Team3-Project#15]
Just starting this so we can keep track of what we haven't finished:
- [x] Add more pellets to maps
- [x] Pacman death
- [x] Big Pellets that make ghosts killable
- [x] Starting a new map when all pellets are eaten.
- [x] Pacman Teleportation

**Comment by alan-boggess** [comment]
"Add more pellets to the map" is effectively handled, now.  (Dots now spawn on TileID3's on map creation.)

**Comment by alan-boggess** [comment]
Teleportation is tentatively implemented.  I'm sure Spence could probably find a case that breaks it, but it's working well enough for a demo.

**Comment by alan-boggess** [comment]
I think I've got a lead on starting a new map, but it involves destroying the current map first, so it might just break everything.

Stay tuned...

**Comment by SpenceSellers** [comment]
Personally I think the easiest way to start a new map is to reload the page entirely. The name of the current map could be (optionally) passed in by a URL parameter, defaulting to the first map.

Going to the next map would just completely reload the whole page.

**Comment by alan-boggess** [comment]
I think maintaining score and lives that way is messier.

**Comment by SpenceSellers** [comment]
Good point, I didn't consider scores and lives. I figured that the next level would start with zero score and zero lives, but I suppose that wouldn't be much fun. That could easily be fixed by two more URL params (the code for this would really be like six lines), but yeah, it does seem like a good way of acquiring technical debt :D

**Comment by alan-boggess** [comment]
Implemented and pushed.

If we want to add more maps to the rotation, we should add their relative path to the static array mapsAssets.

I spent way too little time bug hunting this implementation for it to be working this well.  Please raise an issue when you find something it broke.
